core/extension/gdextension_interface.cpp

```diff
@@ -1475,7 +1475,7 @@ static GDExtensionMethodBindPtr gdextension_classdb_get_method_bind(GDExtensionC
 
 static GDExtensionObjectPtr gdextension_classdb_construct_object(GDExtensionConstStringNamePtr p_classname) {
 	const StringName classname = *reinterpret_cast<const StringName *>(p_classname);
-	return (GDExtensionObjectPtr)ClassDB::instantiate_no_placeholders(classname);
+	return (GDExtensionObjectPtr)ClassDB::instantiate_without_postinitialization(classname);
```

Comment:
I'm not sure we can change `classdb_construct_object` for backwards compatibility reasons - I think we may need to create a `classdb_construct_object2` (with a `2` appended) in `gdextension_interface.h`.

We only recently decided that it was the responsibility of the GDExtension binding to send `NOTIFICATION_POSTINITIALIZE` to instances of extension classes created from within the GDExtension binding (see https://github.com/godotengine/godot-cpp/issues/1269#issuecomment-1813559194). I'm not sure that any other GDExtension binding is actually doing that yet, except for godot-cpp. So, if we changed `classdb_construct_object` to not send that notification, then it won't happen at all for bindings that haven't added their own mechanism to send it, which could break the "owner class".